pkg/controller/certificates/internal/secretsmanager/secret.go

```diff
@@ -203,8 +245,12 @@ func (s *SecretsManager) setValues(crt *cmapi.Certificate, secret *corev1.Secret
 			delete(secret.Data, jksSecretKey)
 			delete(secret.Data, jksTruststoreKey)
 		}
-	}
 
+		// Add additional output formats
+		if utilfeature.DefaultFeatureGate.Enabled(feature.AdditionalCertificateOutputFormats) {
+			updateSecretWithAdditionalOutputFormats(crt, secret, data)
```

Comment:
Check the error.

Reply:
We need to be checking the error returned from `updateSecretWithAdditionalOutputFormats` here.